docs/src/data/hid.js

```diff
@@ -0,0 +1,7835 @@
+/*
+ * Copyright (c) 2020 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: CC-BY-NC-SA-4.0
+ */
+
+import {
+  keyboard as keyboardApplication,
+  consumer as consumerApplication,
+} from "./hid-applications";
+
+import { key as keyPage, consumer as consumerPage } from "./hid-usage-pages.js";
+
+export default [
+  {
+    names: ["A"],
+    description: "a and A",
+    context: "Keyboard",
+    clarify: false,
+    usages: [
+      {
+        application: keyboardApplication,
+        item: (keyPage << 16) | 0x04,
```

Comment:
The bit shifting and binary `OR` here seems duplicated, thoughts on making this something like:

```suggestion
        item: hidExtendedUsage(keyPage, 0x04),
```

to parallel that use in the firmware code?

Reply:
Please check the re-push.

Reply:
Yeah, looks great!